daemon/containerd/image_list.go

```diff
@@ -136,10 +150,12 @@ func (i *ImageService) Images(ctx context.Context, opts types.ImageListOptions)
 		}
 	}
 
+	sort.Sort(sort.Reverse(byCreated(summaries)))
```

Comment:
This looked scary at first; was wondering if it would be sorting _twice_ to reverse it, but looks like `sort.Reverse` just overrides the underlying `sort.Interface` and flips the values.

Slightly wondering though if we always sort `byCreated` in reverse order, or we should update the order there.

Reply:
Comes from https://github.com/moby/moby/blob/master/daemon/images/image_list.go#L251

Reply:
Yup, found that one! Still double checked if it was ok 😂